src/components/Modal/DialogDeleteNote.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import { useDispatch, useSelector } from 'react-redux';
+import { colors } from '../../colors';
+import { deleteNoteAction } from '../../components/BottomBar/BottomBarActions';
+import { makeStyles } from '@material-ui/styles';
+
+const useStyles = makeStyles(() => ({
+  dialogContent: {
+    background: colors.menuBackground,
+  },
+  dialogContentText: {
+    color: colors.textColor,
+  },
+  dialogAction: {
+    background: colors.menuBackground,
+  },
```

Comment:
Is it possible to have all these styles in `src/theme.js` ? Where we have the style for `MuiDialogTitle`.